'make debug' fails

```make debug``` in ```tag: v0.18.1.1```, same for ```tag: v0.18.1.2```

```
[ 31%] Building CXX object contrib/epee/src/CMakeFiles/obj_epee.dir/int-util.cpp.o
[ 31%] Building CXX object contrib/epee/src/CMakeFiles/obj_epee.dir/portable_storage.cpp.o
In file included from /home/monero/contrib/epee/src/portable_storage.cpp:35:
/home/monero/contrib/epee/include/storages/portable_storage_from_bin.h: In member function ‘epee::serialization::storage_entry epee::serialization::throwable_buffer_reader::load_storage_entry()’:
/home/monero/contrib/epee/include/storages/portable_storage_from_bin.h:322:5: error: control reaches end of non-void function [-Werror=return-type]
  322 |     }
      |     ^
/home/monero/contrib/epee/include/storages/portable_storage_from_bin.h: In member function ‘epee::serialization::storage_entry epee::serialization::throwable_buffer_reader::load_storage_array_entry(uint8_t)’:
/home/monero/contrib/epee/include/storages/portable_storage_from_bin.h:231:5: error: control reaches end of non-void function [-Werror=return-type]
  231 |     }
      |     ^
cc1plus: some warnings being treated as errors
make[3]: *** [contrib/epee/src/CMakeFiles/obj_epee.dir/build.make:356: contrib/epee/src/CMakeFiles/obj_epee.dir/portable_storage.cpp.o] Error 1
make[3]: Leaving directory '/home/monero/build/Linux/_HEAD_detached_at_v0.18.1.1_/debug'
make[2]: *** [CMakeFiles/Makefile2:1496: contrib/epee/src/CMakeFiles/obj_epee.dir/all] Error 2
make[2]: Leaving directory '/home/monero/build/Linux/_HEAD_detached_at_v0.18.1.1_/debug'
make[1]: *** [Makefile:146: all] Error 2
make[1]: Leaving directory '/home/monero/build/Linux/_HEAD_detached_at_v0.18.1.1_/debug'
make: *** [Makefile:58: debug] Error 2
```

--- a/contrib/epee/include/storages/portable_storage_from_bin.h
+++ b/contrib/epee/include/storages/portable_storage_from_bin.h
@@ -231,6 +231,7 @@ namespace epee
       default: 
         CHECK_AND_ASSERT_THROW_MES(false, "unknown entry_type code = " << type);
       }
+      return read_ae<int8_t>(); // unreachable, dummy return to avoid compiler warning
     }
 
     inline 
@@ -322,6 +323,7 @@ namespace epee
       default: 
         CHECK_AND_ASSERT_THROW_MES(false, "unknown entry_type code = " << ent_type);
       }
+      return read_se<int8_t>(); // unreachable, dummy return to avoid compiler warning
     }
     inline 
     void throwable_buffer_reader::read(section& sec)

